pkg/kudoctl/packages/install/package.go

```diff
@@ -139,3 +144,22 @@ func validateParameters(instance v1beta1.Instance, parameters []v1beta1.Paramete
 
 	return nil
 }
+
+func updateOperatorTasks(pkgs []Dependency, operatorVersion *v1beta1.OperatorVersion) {
```

Comment:
I'm usually for short names, but in this case, I'd sacrifice brevity for clarity:
```suggestion
func updateKudoOperatorTaskPackageNames(pkgs []Dependency, operatorVersion *v1beta1.OperatorVersion) {
```

and add a comment to the method saying *why* we're doing this